hermeto/core/package_managers/general.py

```
@@ -111,26 +111,24 @@ def download_binary_file(
+    session = _get_pkg_requests_session()
     try:
-        log.debug("requests.get(url: %s)", url)
-        resp = _get_pkg_requests_session().get(
-            url, stream=True, verify=not insecure, auth=auth, timeout=timeout
-        )
-        resp.raise_for_status()
+        response = session.get(url, stream=True, verify=not insecure, auth=auth, timeout=timeout)
+        response.raise_for_status()
     except requests.RequestException as e:
-        raise FetchError(f"Could not download {url}: {e}")
+        raise FetchError(f"Could not download {url}") from e
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)